Source/WebKit/UIProcess/API/Cocoa/WKWebViewInternal.h

```diff
@@ -60,6 +60,10 @@
 #define WK_WEB_VIEW_PROTOCOLS <WKShareSheetDelegate>
 #endif
 
+#if PLATFORM(COCOA)
+#import "WKTextIndicatorStyleType.h"
+#endif
```

Comment:
I don't think you actually need the `#if PLATFORM(COCOA)` here?

If you don't slot it in after `#import "PDFPluginIdentifier.h"`. If it's needed, move this code up before line 55, where the protocol definitions start.